app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="container d-flex flex-column justify-content-center align-items-center w-50 mt-4">
+<h2>Sign up</h2>
+<div class="'d-flex flex-column w-100">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, class: 'w-100' %>
+  </div>
+  
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'w-100' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: 'w-100' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'w-100' %>
+  </div>
```

Comment:
- [ ] Good job so far guys 🔥 💪🏾 It looks like there is something wrong with your authentication features 😢 I couldn't be able to create account and therefore I couldn't login. Probably there is something wrong with the validation `tokens` can't be black. I'd suggest that you investigate more on your user validations 🙏🏾 
<img width="752" alt="Screen Shot 2023-09-16 at 01 01 36" src="https://github.com/Nahid1911/RecipeApp-RoR/assets/90222110/51cd154b-9cd2-49bc-ba46-4fde5ceea4c2">
<img width="1385" alt="Screen Shot 2023-09-16 at 01 04 19" src="https://github.com/Nahid1911/RecipeApp-RoR/assets/90222110/b4191fe6-e81f-43c5-bf59-fe2941dbb2c6">
<img width="1414" alt="Screen Shot 2023-09-16 at 01 05 03" src="https://github.com/Nahid1911/RecipeApp-RoR/assets/90222110/c4ae094e-9c0e-4aac-aa40-8000f63925d9">

Reply:
Issue resolved